exonum-java-binding/launcher-plugins/tests/test_data/custom_message_name.yml

```diff
@@ -0,0 +1,26 @@
+networks:
+  - host: "127.0.0.1"
+    ssl: false
+    public-api-port: 8080
+    private-api-port: 8081
+
+plugins:
+  artifact:
+    cryptocurrency: "exonum_java_instance_plugin.JavaInstanceSpecLoader"
```

Comment:
Where is the 'cryptocurrency' artifact used? I only see timestamping. Isn't it redundant?